PyInstaller/depend/bindepend.py

```diff
@@ -622,7 +622,7 @@ def _getImports_macholib(pth):
             #    '../lib\x00\x00')
             cmd_type = command[0].cmd
             if cmd_type == LC_RPATH:
-                rpath = str(command[2])
+                rpath = command[2].decode('ascii')
```

Comment:
I doubt using "ascii" here is the right thing to do. Esp. since AFAIK the std-encoding on OS X is UTF-8.